plugin/pulpcore/plugin/download/http.py

```diff
@@ -1,8 +1,34 @@
+import logging
+
 import aiohttp
+import backoff
 
 from .base import attach_url_to_exception, BaseDownloader, DownloadResult
 
 
+log = logging.getLogger(__name__)
+
+
+logging.getLogger('backoff').addHandler(logging.StreamHandler())
+
+
+def inspect_to_retry(exc):
+    """
+    Inspect a raised exception to determine if a retry is needed
```

Comment:
For the same reason as above, probably best to be clear about what the function is doing in the description.

```Inspect a raised exception to determine if the download should not be retried```

It's kind of a double negative and not very ergonomic but, eh, that's what the library does.

Reply:
I've updated this language some in my next push to match the renaming of the function. I think it's an improvement, but we can see what you think.